VideoLocker/assets/COURSE_NOT_LISTED.htm

```diff
@@ -439,8 +439,8 @@
 
 <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
 style='mso-bookmark:OLE_LINK6'><span style='font-size:11.0pt;font-family:Arial;
-mso-bidi-font-family:Arial'>We're working hard to make all of our courses mobile-friendly. If you can't find a course,
- check back soon. We're adding more courses every day!</span></span></p>
+mso-bidi-font-family:Arial'>We�re working hard to make all of our courses mobile-friendly. If you can�t find a course,
+ check back soon. We�re adding more courses every day!</span></span></p>
```

Comment:
I don't know why it's showing up weird, but you can just use `&rsquo;` here instead.